tests/unit/unit_test_client.py

```diff
@@ -821,3 +821,146 @@ def test_update(self):
         self.mock_restPUT.assert_called_once_with('/projectSettings', body=json.dumps(new_location))
         self.mock_restPOST.assert_not_called()
 
+
+class TestMembershipInvitation:
```

Comment:
as a general style, test names should be:
`test_<function/method name>__<case being tested>` 
to make it easier to identify the function/method under test
the double underscore helps to separate the name from being confused with the case being tested